docs/sources/api/_index.md

```diff
@@ -209,37 +217,37 @@ $ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query=
   }
 }
 ```
+Set the `x-scope-orgid` header,
+as defined in [Grafana Loki Multi-Tenancy](../operations/multi-tenancy/),
+to query specific tenants.
+Here is the same example query applied to the single tenant called `Tenant1`:
```

Comment:
For some reason I don't like the word applied but I can't come up with a better alternative so I think its fine. I guess I think of the query as being 'directed at' that tenant, rather I think of 'applied' as more referring to something more permanent (e.g., you apply a setting or apply a configuration change).

Reply:
Maybe
```
Here is the same example query for the single tenant called `Tenant1`:
```